### PR TITLE
Added recipe for JasPer 2.0.10. Recent versions use CMake rather than…

### PR DESCRIPTION
**Pull request by mboisson**
… Configure

The compiled paths are also different (lib/libjasper.a => lib64/libjasper.so).

### CODE REVIEW

File: easybuild/easyconfigs/j/JasPer/JasPer-2.0.10-intel-2016b.eb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+easyblock = 'CMakeMake'
+
+name = 'JasPer'
+version = '2.0.10'
+
+homepage = 'http://www.ece.uvic.ca/~frodo/jasper/'
+description = """The JasPer Project is an open-source initiative to provide a free
+ software-based reference implementation of the codec specified in the JPEG-2000 Part-1 standard."""
+
+separate_build_dir = True
```
Comment (boegel) [inline easybuild/easyconfigs/j/JasPer/JasPer-2.0.10-intel-2016b.eb]: @mboisson: please move this down below `source_urls` (with a blank line in between)

Why is this needed by the way? Isn't the `make install`doing the proper thing here?

Comment (mboisson) [inline easybuild/easyconfigs/j/JasPer/JasPer-2.0.10-intel-2016b.eb]: Because JasPer's CMake config forbids building in the source directory. It was either specifying this option, or patching JasPer's CMake config.

Comment (boegel) [inline easybuild/easyconfigs/j/JasPer/JasPer-2.0.10-intel-2016b.eb]: Ah, sorry, of course, I was mixing this up with `build_in_installdir` for some reason...

Ignore my rant, but please do move it down as requested.

Comment (mboisson) [inline easybuild/easyconfigs/j/JasPer/JasPer-2.0.10-intel-2016b.eb]: Done

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node2060.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/d325038dc1ad1087fc3e061905d94262 for a full test report.

**Comment by boegel** [comment]
@mboisson missing CMake build dependency...

**Comment by mboisson** [comment]
Arg... this is what I get for developing with nix-provided stuff.  Fixed.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2116.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/b2f7391f644219544159c77860ba815d for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2494.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/13318c96d14d76ed70cb8e7967ed35d3 for a full test report.

**Comment by boegel** [comment]
lgtm, going in when Travis is happy (seems like Travis is unstable atm, cfr. https://www.traviscistatus.com/, will check back later)

**Comment by boegel** [comment]
Going in, thanks @mboisson!
